exporter/splunkhecexporter/README.md

```diff
@@ -43,6 +43,8 @@ The following configuration options can also be configured:
   data to be dropped instead. Applicable in the `logs` pipeline only.
 - `profiling_data_enabled` (default: true): Specifies whether the profiling data is exported. Set it to `false` if 
   you want the profiling data to be dropped instead. Applicable in the `logs` pipeline only.
+- `health_path` (default = '/services/collector/health'): The path reporting [health checks](https://docs.splunk.com/Documentation/Splunk/9.0.1/RESTREF/RESTinput#services.2Fcollector.2Fhealth).
+- `health_check_enabled` (default = false): Whether to perform Splunk HEC Health Check during the exporter's startup.
```

Comment:
```suggestion
- `health_check_enabled` (default = false): Whether to check health of the receiving endpoint during execution. If this check is enabled enabled, the exporter will abort sending data and use the retry policy of the pipeline to retry at a later time.
```

Reply:
`If this check is enabled enabled, the exporter will abort sending data and use the retry policy of the pipeline to retry at a later time` 

this is how it worked initially, but after @dmitryax suggestion above:

> > > Alternatively, we could put that HEC Health check in Validate method so that the otel process won't even start if Splunk does not respond
> > 
> > 
> > That seems to be a better option, but in `createExporter` not `Validate`. `Validate` shouldn't make any external calls
> 
> Noted

I changed it in a way that it's not retrying, it performs one check in `createExporter` and if health check fails, it returns the message:
```
Error: cannot build pipelines: failed to create "splunk_hec" exporter, in pipeline "logs": health check failed: Get "https://10.202.4.254:8088/services/collector/health": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
2022-12-05 10:41:15.969534 I | collector server run finished with error: cannot build pipelines: failed to create "splunk_hec" exporter, in pipeline "logs": health check failed: Get "https://10.202.4.254:8088/services/collector/health": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
```
and doesn't start at all.